client/header/activity-panel/test/index.js

```diff
@@ -193,4 +197,72 @@ describe( 'Activity Panel', () => {
 
 		expect( queryByText( 'Store Setup' ) ).toBeDefined();
 	} );
+
+	describe( 'help panel tooltip', () => {
+		it( 'should render highlight tooltip when trackedStartedTasks includes task, task is not completed, and tooltip not shown yet', () => {
+			const { queryByText } = render(
+				<ActivityPanel
+					requestingTaskListOptions={ false }
+					taskListComplete={ false }
+					taskListHidden={ false }
+					trackedStartedTasks={ [ 'payment', 'payment' ] }
```

Comment:
This doesn't seem to match the new format of being an object. Are these tests passing by accident?

Reply:
Good catch, turns out `tobeDefined` checks if it is not `undefined`.
I used `getByText` and `toBeInTheDocument` instead, which correctly fails if it isn't shown.

Also updated the data format to match the new layout.